packages/oauth/src/common.test.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import { readLocal, removeLocal, saveLocal } from "./common.js";
+
+describe("local functions", () => {
+  beforeEach(() => {
+    vi.stubGlobal("localStorage", {
+      setItem: vi.fn(),
+      removeItem: vi.fn(),
+      getItem: vi.fn(),
+    });
+  });
+
+  it("should use the old style keys when there is no refresh token marker", () => {
+    const client = {
+      client_id: "hi_mom",
+      $refreshTokenMarker: undefined,
+    };
+
+    const oldLocalStorageKey = `@osdk/oauth : refresh : ${client.client_id}`;
```

Comment:
Ensure that people not using this feature won't get logged out on upgrade of `@osdk/oauth`